addons/wowl/static/src/components/notification/notification.scss

```diff
@@ -18,4 +18,30 @@
       }
     }
   }
+
+  .o_toast {
+    max-width: 320px;
+    overflow: hidden;
+    font-size: 1.08333333rem;
+    background-color: rgba(255, 255, 255, 0.7);
+    background-clip: padding-box;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    box-shadow: 0 0.25rem 0.75rem rgba(0, 0, 0, 0.1);
+    backdrop-filter: blur(10px);
+    border-radius: 0.25rem;
+  }
+
+}
+
+.fade-enter-active,
+.fade-leave-active {
+  transition: all 0.5s;
+}
+
+.fade-enter {
+  opacity: 0;
+}
+
+.fade-leave-to {
+  opacity: 0;
```

Comment:
you should define a mixin for this (with the delay as param), and use it on `.notification`